### PR TITLE
IDEX-3415: download source jars

### PR DESCRIPTION
**Pull request by svor**

### CODE REVIEW

File: plugin-java/che-plugin-java-ext-jdt/che-jdt-ext-machine/src/main/java/org/eclipse/che/jdt/rest/JavaClasspathService.java [hunk 1]
```diff
@@ -65,28 +65,42 @@ public boolean update(@QueryParam("projectpath") final String projectPath) throw
...
     }
 
-    private boolean generateClasspath(final String projectPath, File dir) {
+    private boolean dependencyUpdateProcessor(final String projectPath, File dir) {
+        String command = MavenUtils.getMavenExecCommand();
+
+        ProcessBuilder classPathProcessBuilder = new ProcessBuilder().command(command,
+                                                                              "dependency:build-classpath",
+                                                                              "-Dmdep.outputFile=.codenvy/classpath.maven")
+                                                                     .directory(dir)
```
Comment (skabashnyuk) [inline plugin-java/che-plugin-java-ext-jdt/che-jdt-ext-machine/src/main/java/org/eclipse/che/jdt/rest/JavaClasspathService.java]: Can you describe what is ".codenvy"?
why not .che?

Comment (svor) [inline plugin-java/che-plugin-java-ext-jdt/che-jdt-ext-machine/src/main/java/org/eclipse/che/jdt/rest/JavaClasspathService.java]: This is a folder which contains configuration files (classpath, modules). 
.codenvy will be renamed as part of issue https://jira.codenvycorp.com/browse/IDEX-3480.

Comment (skabashnyuk) [inline plugin-java/che-plugin-java-ext-jdt/che-jdt-ext-machine/src/main/java/org/eclipse/che/jdt/rest/JavaClasspathService.java]: ок

**Comment by vparfonov** [comment]
ok

**Pull request merged**